.github/ISSUE_TEMPLATE/new-scraper.md

```diff
@@ -1,16 +1,24 @@
 ---
-name: New scraper
-about: Add a new scraper
-title: Scraper for [LOCATION_HERE]
+name: "📥 New scraper source"
```

Comment:
I felt like this one is someone telling us "here's where you can get data" and that this made more sense